Xamarin.Forms.Platform.Tizen/Renderers/EntryRenderer.cs

```diff
@@ -19,6 +19,7 @@ public EntryRenderer()
 			RegisterPropertyHandler(Entry.PlaceholderProperty, UpdatePlaceholder);
 			RegisterPropertyHandler(Entry.PlaceholderColorProperty, UpdatePlaceholderColor);
 			RegisterPropertyHandler(InputView.MaxLengthProperty, UpdateMaxLength);
+			RegisterPropertyHandler(Entry.IsReadOnlyProperty, UpdateIsReadOnly);
```

Comment:
I think the whitespace preceeding the if was lost here.

Reply:
You are right, I will fix that also :)